index.js

```diff
@@ -33,6 +33,11 @@ const getConfig = () => {
         config.tags = process.env.LOGDNA_TAGS.split(',').map(tag => tag.trim()).join(',');
     }
 
+    if (process.env.RAW_EVENT_MESSAGE) {
+        config.raw_event_message = process.env.RAW_EVENT_MESSAGE.toLowerCase();
+        config.raw_event_message = config.raw_event_message.indexOf('yes') > -1 || config.raw_event_message.indexOf('true') > -1;
```

Comment:
This will also mean `RAW_EVENT_MESSAGE` can be enabled with values like `yesno` and `true213213123` which doesn't seem right.

Reply:
@jakedipity, well, as documented, if the customer wants to enable, this option should be properly used; so, it should be either `yes` or `true` (both case insensitive) to enable that option. It is up to the customer who wants to see things work as wanted.

Reply:
So what if it's improperly set? The current code will accept an improper value as valid. It doesn't make a whole lot of difference, but it does give the appearance (I think) of a shoddy implementation. I won't block on this but I think it can be better - and better in this case might be very subjective.

Reply:
@jakedipity, you are right! Updated it to be stricter on checking the values